components/camel-spring-xml/pom.xml

```diff
@@ -137,6 +137,11 @@
             <artifactId>derby</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <scope>test</scope>
```

Comment:
spring-xml uses JAXB as parser so I would assume the JAXB annotations are required, not test scoped

Reply:
That is the same for camel-cdi as it also has a XML that is loaded via JAXB.
And so does camel-blueprint in camel-karaf project

Reply:
Thanks. I added them in the test scope because it was test code that was failing, but I'll change it to the default one.